tests/unittests/BackendTestUtils.cpp

```diff
@@ -310,6 +310,7 @@ void compareAgainstInterpreter(llvm::StringRef backendName,
   assert(resultTensors.size() == count &&
          "Should get the same number of Tensors back as count.");
 
+  cctxB.optimizationOpts.enableTypeDemotion = false;
```

Comment:
Why is this necessary? Maybe add comment?

Reply:
I think originally I wanted to keep it same as Interpreter since there it's disabled, but on more thought I don't think it's necessary. Removed